appeals/web/src/server/lib/mappers/page-components/interested-party-comments/view-and-review/reject.mapper.js

```diff
@@ -0,0 +1,92 @@
+import { appealShortReference } from '#lib/appeals-formatter.js';
+
+/** @typedef {import("../../../../../appeals/appeal-details/appeal-details.types.js").WebAppeal} Appeal */
+/** @typedef {import("../../../../../appeals/appeal-details/interested-party-comments/interested-party-comments.types.js").Representation} Representation */
+/** @typedef {import("../../../../../appeals/appeal-details/interested-party-comments/interested-party-comments.types.js").RepresentationRejectionReason} RepresentationRejectionReason */
+/** @typedef {import("../../../../../appeals/appeal-details/interested-party-comments/interested-party-comments.types.js").RejectionReasonUpdateInput} RejectionReasonUpdateInput */
+/** @typedef {import("../../../../../appeals/appeal-details/interested-party-comments/interested-party-comments.types.js").RejectionReasons} RejectionReasons */
```

Comment:
Alex made me change a bunch of these to absolute imports on one of my recent PRs. I'm not that bothered if you do or not but if you want to you can swap all the `../`s for a `#`

Reply:
this file should have been removed, possibly bad rebase